### PR TITLE
add standards

### PR DESCRIPTION
**Pull request by MylesBorins**
Would be good to generate meetings for standards team

Meets every other Tuesday. Need to get added to cal.

### CODE REVIEW

File: templates/meeting_base_standards [hunk 1]
```diff
@@ -0,0 +1,25 @@
+CALENDAR_FILTER="OpenJS Standards Team Meeting"
+CALENDAR_ID="nodejs.org_nr77ama8p7d7f9ajrpnu506c98@group.calendar.google.com"
```
Comment (mhdawson) [inline templates/meeting_base_standards]: I doubt this is the right Calendar ID as I expect the calendar should be the one for the OpenJS foundation.

Comment (mhdawson) [reply templates/meeting_base_standards]: hmm, I see it is below.  So line above is duplicate and should be removed.

**Comment by mhdawson** [comment]
@MylesBorins  also seems to be missing the template for the minutes

**Comment by MylesBorins** [comment]
Should be fixed

**Comment by mhdawson** [comment]
@MylesBorins when is the next meeting?  I need an entry on the calendar to be able to test.

**Comment by sendilkumarn** [comment]
The next meeting is scheduled to be on Jul 16th.

**Comment by mhdawson** [comment]
@sendilkumarn I don't see it on the OpenJS Calendar.  I need that entry to test out the automation as the issues/minutes are generated off the calendar entries.

**Comment by sendilkumarn** [comment]
Oops sorry, I thought @MylesBorins already added that entry in the OpenJSF calendar.

**Comment by mhdawson** [comment]
@MylesBorins can you let me know when you add the calendar entry.

**Comment by MylesBorins** [comment]
@mhdawson done

**Comment by mhdawson** [comment]
Ran automation to generated test artifacts.  Looks good.

Should auto generate artifacts for next meeting on Friday (2 days from now).

**Pull request merged**